### ISSUE TITLE
Somtimes new change can't build unless redeploy.

### ISSUE THREAD

**Issue by chiperman** [transitive-bullshit/nextjs-notion-starter-kit#269]
When I change some content or add a new page, it will not appear unless I redeploy in the vercel. Can anyone know why? Is it beacuse I have many  linked database or pages in the program?

**Comment by transitive-bullshit** [comment]
Hey @chiperman, as always, please include a notion ID and/or public notion site to test with. There are so many things that could possibly be going on, that without a reproduction example, it's difficult for me to give a constructive response.

New deployments will for sure always update the latest content. Otherwise, the default is to use next.js's ISSR to regenerate pages in the background once every X seconds. If that fails for any reason (like the renderer throwing an error or the notion API returning something unexpected), then next.js will use the most recent version of that page.

Make sure to check your Vercel deployment logs to see what's going on when refreshing a page to make sure that it's actually getting the latest version and that there are no unexpected errors.

**Comment by chiperman** [comment]
@transitive-bullshit This is my open site: https://notion.chiperman.tk and the notion ID is : f7798e49ceb84f7c89931e85a92af60b, and I have just added this git-command page to the blog-post database. But I can't see it unless I try to redeploy it in the Vercel. It won't appear, I was wondering if this problem is caused by the fact that I have too many associated databases in the whole project.
<img width="322" alt="截屏2022-04-19 15 13 17" src="https://user-images.githubusercontent.com/34889778/163946580-e9e25111-5e45-4985-ad2c-0b9e98dff1ac.png">

**Comment by Kevin-Bian21** [comment]
Yes, I have the same problem. When I add or change content in the note, I need to manually redeploy it in vercel to make it effective。

**Comment by Krystal-zqy** [comment]
I have the same problem.

**Comment by PersistVenture** [comment]
I also have same issues. :(

**Comment by PersistVenture** [comment]
@chiperman hey! have you resolved the issue? If so, can you please share what steps you did to resolve the issue.

Thanks.

**Comment by chiperman** [comment]
@PersistVenture Sorry, I still have this problem and I think maybe we have too much include pages in the the project cause when I remove everything and try to add it at a once it will work properly. But once reach a point, it will appear the same question.

**Comment by thefinnomenon** [comment]
Same issue

**Comment by nooobcoder** [comment]
Same issue @transitive-bullshit 
I also tweaked the 
`revalidate: 30` to build after 30 secs, but no luck

**Comment by 3chospirits** [comment]
Same issue here as well

**Comment by fantasyEthan** [comment]
I have the same problem as well. Wonder if it is the problem of network.

**Comment by kevinjalbert** [comment]
I think the suggestion in https://github.com/transitive-bullshit/nextjs-notion-starter-kit/issues/319 fixes this issue.

I was unable to get new pages to show up, but with this everything works now.

**Comment by sivannavis** [comment]
Hi! I waited long enough and it's still not synced. Is it like an automatic redeploy on Vercel or what? Any idea to make the sync easier and faster?

**Comment by breezjin** [comment]
Same issue here as well.

**Comment by muhamad-rizki** [comment]
![image](https://user-images.githubusercontent.com/15137312/204706661-18f93c74-35e8-4a86-89b1-0100589ea455.png)

I got this error when trying to open recently created page (after deploy) if I try to redeploy the server, it turns to be fine

**Comment by Nexus-Gits** [comment]
new change can't build unless redeploy. Also Home page post images not updated. but other updates show without redploy. but new post & home page cover images not update automatically.
